unittests/ProfilerTest.cpp

```diff
@@ -0,0 +1,41 @@
+#include "util/Profiler.h"
```

Comment:
I think we always use `#include <headerfile>` style everywhere (except two places that should probably be fixed too).